dask_planner/src/expression.rs

```diff
@@ -633,6 +637,24 @@ impl PyExpr {
         }
     }
 
+    #[pyo3(name = "getIntervalDayTimeValue")]
```

Comment:
We should also add methods to extract this information from interval yearMonth and MonthDayNano types